containerfiles/Containerfile

```diff
@@ -0,0 +1,27 @@
+FROM fedora:$TAG
+
+ARG TAG
+ARG NAME
+
+RUN dnf -y update
+RUN dnf -y install 'dnf-command(copr)'
+
+# enable nightly repositories
+RUN if [ "$NAME" == "dnf5-nightly" ]; then \
+        dnf copr -y enable rpmsoftwaremanagement/dnf-nightly ; \
+    fi
+
+# enable testing repositories
+RUN if [ "$NAME" == "dnf5-testing" ] || [ "$NAME" == "dnf5-testing-nightly" ]; then \
+        dnf copr -y enable rpmsoftwaremanagement/"$NAME" ; \
+    fi
+
+# install all the DNF5 stack
+RUN dnf -y install dnf5 dnf-data dnf5-plugins
+
+# clean installed packages
+# from this on we need to be careful to specify which dnf to use
+RUN dnf4 -y remove 'dnf-command(copr)'
+#clean cache
+RUN dnf4 clean all
+RUN dnf5 clean all
```

Comment:
These cleanup actions need to be "squashed" into a single RUN command (single layer) otherwise the only effect is that it doesn't clean up anything (but it creates yet another layer).

Reply:
oh right

Reply:
Actually :-) it seems that all the RUN commands should be merged into a single one here, looking like:
```
RUN dnf -y update \
    && dnf -y install 'dnf-command(copr)' \
    && if [ "$NAME" == "dnf5-nightly" ]; then \
        dnf copr -y enable rpmsoftwaremanagement/dnf-nightly ; fi \
    && ...
```

Reply:
I could do that too. thanks

Reply:
now that I think about it, I don't really know what's the recommended way of building containers, like, what options should be squashed together 🤔 is it ok now?